### DISCUSSION TITLE
contents/achievements.tex如何是作者信息加粗bold? \textbf无效

### DISCUSSION THREAD

**Discussion by PhdBoLi** [sjtug/SJTUThesis#928] [提问] [answered]
\begin{achievements}

\subsection*{学术论文}

\begin{bibliolist}{00}
  \item \textbf{Chen H}, Chan C~T. Acoustic cloaking in three dimensions using acoustic metamaterials[J]. Applied Physics Letters, 2007, 91:183518.
  \item Chen H, Wu B~I, Zhang B, et al. Electromagnetic Wave Interactions with a Metamaterial Cloak[J]. Physical Review Letters, 2007, 99(6):63903.
\end{bibliolist}

**Comment by LogCreative** [comment, marked as answer]
目前测试下来加粗是正常的，检查一下是否正常编译。

**Reply by PhdBoLi** [reply]
> 目前测试下来加粗是正常的，检查一下是否正常编译。

哦？那就很奇怪了，可以正常编译，只是用windows平台的Texlive2022/2023的环境，这个是否有影响呢？

**Reply by AlexaraWu** [reply]
有没有可能是盲审模式的原因？你修改的是 `bibliolist` 还是 `bibliolist*`？

**Reply by PhdBoLi** [reply]
不太情况，可能是版本或环境的原因，我又重新下载了4月份的最新版，编译下就没这个问题
